CondTools/Ecal/python/copyBadStrip_cfg.py

```diff
@@ -39,11 +40,12 @@
     IsDestDbCheckedInQueryLog=cms.untracked.bool(True),
     SinceAppendMode=cms.bool(True),
     Source=cms.PSet(
-     firstRun = cms.string('120000'),
+     firstRun = cms.string('200000'),
      lastRun = cms.string('10000000'),
      OnlineDBSID = cms.string('cms_omds_lb'),
-     OnlineDBUser = cms.string('cms_ecal_conf'),
-     OnlineDBPassword = cms.string('*************'),
+#     OnlineDBSID = cms.string('cms_orcon_adg'),  test on lxplus
+     OnlineDBUser = cms.string('cms_ecal_r'),
+     OnlineDBPassword = cms.string('3c4l_r34d3r'),
```

Comment:
yea, seeing that its now easy to find via github searches,  its probably time to change this password now..

On Aug 2, 2016, at 9:26 AM, Marco Musich <notifications@github.com<mailto:notifications@github.com>> wrote:


In CondTools/Ecal/python/copyBadStrip_cfg.py<https://github.com/cms-sw/cmssw/pull/15319#discussion_r73105810>:

>       lastRun = cms.string('10000000'),
>       OnlineDBSID = cms.string('cms_omds_lb'),
> -     OnlineDBUser = cms.string('cms_ecal_conf'),
> -     OnlineDBPassword = cms.string('*************'),
> +#     OnlineDBSID = cms.string('cms_orcon_adg'),  test on lxplus
> +     OnlineDBUser = cms.string('cms_ecal_r'),
> +     OnlineDBPassword = cms.string('3c4l_r34d3r'),


@depasse<https://github.com/depasse> I don't think putting the ecal account password in clear in the release code is good practice

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/cms-sw/cmssw/pull/15319/files/4b5d7b61df01cfc342437343866f92078abccab0#r73105810>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AEzyw-96SNcd6L4kebJJ7a31A6Asdix2ks5qbvETgaJpZM4JXVmQ>.